src/components/Quiz/Quiz.tsx

```diff
@@ -0,0 +1,196 @@
+import { useState, useEffect } from "react";
+import axios from "axios";
+import "./Quiz.css";
+import { useQuiz } from "../../contexts/Quiz-Context";
+import { Modals } from "../Modal/Modal";
+import { Timer } from "../Timer/Timer";
+import { useParams } from "react-router-dom";
+import React from "react";
+import { createStyles, Theme, makeStyles } from "@material-ui/core/styles";
+import Chip from "@material-ui/core/Chip";
+import BubbleChartIcon from "@material-ui/icons/BubbleChart";
+import Button from "@material-ui/core/Button";
+import Loader from "react-loader-spinner";
+
+export function Quiz() {
+  const { id } = useParams();
+  const { state, dispatch } = useQuiz();
+
+  useEffect(() => {
+    (async function () {
+      try {
+        const response = await axios.get(
+          `https://Quiz-App-API.sayuk.repl.co/${id}`
```

Comment:
You can create an `api.js` file, which will have your API URLs
so that if you want to change the endpoint you can do it 
in one place and can avoid repetition

Reply:
I thought of it initially, but as only one URL is being fetched, I think moving it to `api.js` wouldn't be that effective.
Your thoughts on this, please!

Reply:
Make Sense! It's fine as of now.